### DISCUSSION TITLE
Is there some ETA for the next mongoid release?

### DISCUSSION THREAD

**Discussion by cperezabo** [mongodb/mongoid#5833] [Q&A] [answered]
Hi, I need a fix that is already in `master` but I've haven't found some roadmap or something like that to know how much time do I need to wait so I can update my gems.

**Comment by jamis** [comment, marked as answer]
It'll be a few weeks, at least -- we don't have any immediate plans for the next release, but I'll add a ticket for it. We'll discuss a timeline for it at our next standup.

Note that you can lock mongoid to a github repository via your Gemfile, too. It's not generally recommended to run directly on the master branch (it can be unstable, though we do our best to keep it stable), but it might let you use the fix you need while you wait for the gem to be released:

```ruby
gem 'mongoid', git: 'https://github.com/mongodb/mongoid.git'
```

Additionally, for safety, you can specify the specific commit that you want to lock to, if you know a particular commit works reliably for you:

```ruby
gem 'mongoid', git: 'https://github.com/mongodb/mongoid.git', ref: 'd58d8e1'
```

**Reply by cperezabo** [reply]
Yes, I knew I could do that, I just wanted to know whether to do that or to wait, but you already answered me! Thanks

**Comment by alexbevi** [comment]
Noting that [9.0.1](https://github.com/mongodb/mongoid/releases/tag/v9.0.1) was released yesterday (assuming that's why this was closed)
